src/main/java/com/tabnine/binary/BinaryRequestFacade.java

```diff
@@ -38,16 +39,21 @@ public BinaryRequestFacade(BinaryProcessRequesterProvider binaryProcessRequester
             return result;
         } catch (TimeoutException e) {
             binaryProcessRequesterProvider.onTimeout();
+            Sentry.captureException(e);
         } catch (ExecutionException e) {
             if (e.getCause() instanceof TabNineDeadException) {
                 binaryProcessRequesterProvider.onDead(e.getCause());
             }
 
-            Logger.getInstance(getClass()).warn("Tabnine's threw an unknown error during request.", e);
+            String message = "Tabnine's threw an unknown error during request.";
+            Logger.getInstance(getClass()).warn(message, e);
```

Comment:
I think you can remove the log if we send it to sentry? Is there a way to make sentry write to the logs if he has no connection to the internet?